src/application/estimator/estimator.c

```diff
@@ -1 +1,75 @@
 #include "application/estimator/estimator.h"
+#include "FreeRTOS.h"
+#include "application/controller/controller.h"
+#include "stm32h7xx_hal.h"
+#include <queue.h>
+#include <semphr.h>
+
+QueueHandle_t imu_data_handle = NULL;
+estimator_imu_measurement_t imu_data_buffer;
```

Comment:
This is of type `estimator_imu_measurement_t ` but `estimator_update_inputs_imu` takes a pointer to `estimator_all_imus_input_t`. I am pretty sure this also needs to be an instance of `estimator_all_imus_input_t`